command/v7/rollback_command.go

```diff
@@ -14,11 +14,12 @@ import (
 type RollbackCommand struct {
 	BaseCommand
 
-	Force           bool          `short:"f" description:"Force rollback without confirmation"`
-	RequiredArgs    flag.AppName  `positional-args:"yes"`
-	Version         flag.Revision `long:"version" required:"true" description:"Roll back to the specified revision"`
-	relatedCommands interface{}   `related_commands:"revisions"`
-	usage           interface{}   `usage:"CF_NAME rollback APP_NAME [--version VERSION] [-f]"`
+	Force           bool                    `short:"f" description:"Force rollback without confirmation"`
+	RequiredArgs    flag.AppName            `positional-args:"yes"`
+	Strategy        flag.DeploymentStrategy `long:"strategy" description:"Deployment strategy can be canary or rolling. When not specified, it defaults to rolling."`
```

Comment:
**question/non-blocking** Just to understand, this default behaviour is defined in CAPI right not the CLI?

Reply:
Nevermind, saw we had existing default behaviour to rolling